hw/xbox/xbox.c

```diff
@@ -52,7 +52,8 @@
 
 #define MAX_IDE_BUS 2
 
-// XBOX_TODO: Should be passed in through configuration
+int eeprom_fileDescriptor;
```

Comment:
Naming convention is not very consistent. By the way *should* we keep the file open always? If yes do we need to write the whole buffer every time?

Reply:
Oky sound good.